### DISCUSSION TITLE
Configure Copilot to only show suggestions by hotkey

### DISCUSSION THREAD

**Discussion by ivanbrykov** [community/community#21053] [Copilot Conversations] [answered]
I would like to have full control over when Copilot shows inline suggestions (in VSCode). There is a hotkey (option+\\) that triggers a suggestion. Can I configure Copilot to not show any suggestions automatically unless I pressed that hotkey?

**Comment by Daniel-Wedemeyer** [comment, marked as answer]
As far as I know, this has been solved for [VS Code](https://stackoverflow.com/questions/71224911/can-github-copilot-stop-auto-suggesting-instead-be-triggered-by-a-keystroke), but not Visual Studio. I would really like to have this option in VS.

**Reply by 0x-ximon** [reply]
That solution seems to disable Copilot altogether as the keystroke no longer brings up suggestions. It didn't work for me.

**Reply by ivanbrykov** [reply]
it actually worked for me! Thank you!

**Comment by whoami-anoint** [comment]
You can press ctrl + enter, copilot usually suggest upto 10 solutions at a go, once you click that, you can accept anyone that fit your intent.
